packages/core/integration-tests/test/html.js

```diff
@@ -2322,8 +2322,16 @@ describe('html', function() {
   });
 
   it('should work with bundle names that have colons in them', async function() {
+    // Windows paths cannot contain colons, so write the file here (in memory).
+    await overlayFS.mkdirp(path.join(__dirname, 'integration/url-colon'));
```

Comment:
Does this fix the colon issue? I notice the benchmarks still fail due to a invalid path with colons and I notice it on[ other PRs as well](https://github.com/parcel-bundler/parcel/pull/6514). Is that just a bug we introduced somewhere and is now in V2 ? 😟

Reply:
it should but it seems the test is still failing on windows. I will look into it